src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -2567,6 +2646,22 @@ class Simplex_tree {
     }
   }
 
+  std::size_t num_simplices_and_filtration_size(Siblings const* sib, std::size_t& fv_byte_size) const {
```

Comment:
Could the name reflect the fact that this is about serialization? "size" is not very specific.

Reply:
What do you think of `num_simplices_and_filtration_serialization_size`?

Reply:
That's fine (it is private, it doesn't need to be pretty).